e2e/setup/application.yml

```diff
@@ -1,13 +1,21 @@
 # Template configuration for the e2e tests
 hedera:
-  network_type:
+  network_type: testnet
+  db_validation:
+    host: 127.0.0.1
+    name: hedera_validator
+    password: hedera_db_validation_pass
+    port: 5432
+    username: hedera_db_validation
```

Comment:
Can we clear the actual data?

Reply:
There is a documentation file `docs/testings.md`, which will need to be updated.